apiserver/common/networkingcommon/types.go

```diff
@@ -179,7 +179,7 @@ func NetworkInterfacesToStateArgs(devs corenetwork.InterfaceInfos) (
 		}
 		addr, err := networkAddressToStateArgs(dev, dev.PrimaryAddress())
 		if err != nil {
-			logger.Warningf("ignoring address for device %q: %v", dev.InterfaceName, err)
+			logger.Warningf("ignoring address for device %+v: %v", dev, err)
```

Comment:
Does the `%+v` print out something nice, or should we have a `String()` on the type?